src/test/java/com/bhoruka/bloodbank/dao/integration/DynamoDbLocalTest.java

```diff
@@ -25,15 +25,15 @@ public void test() {
             listTables(dynamodb.listTables(), "DynamoDB Embedded");
         } finally {
             // Shutdown the thread pools in DynamoDB Local / Embedded
-            if(dynamodb != null) {
+            if (dynamodb != null) {
```

Comment:
Is putting a space before/after braces necessary?

Reply:
yes, it's part of our checkstyle which is based on google_checks. It makes the code more readable. More info here: https://checkstyle.sourceforge.io/google_style.html